flink-connectors/flink-connector-pulsar/src/main/java/org/apache/flink/connector/pulsar/source/config/SourceConfiguration.java

```diff
@@ -79,6 +82,8 @@ public SourceConfiguration(Configuration configuration) {
         this.subscriptionType = get(PULSAR_SUBSCRIPTION_TYPE);
         this.subscriptionMode = get(PULSAR_SUBSCRIPTION_MODE);
         this.allowKeySharedOutOfOrderDelivery = get(PULSAR_ALLOW_KEY_SHARED_OUT_OF_ORDER_DELIVERY);
+        this.enableMetrics =
+                !get(PULSAR_ENABLE_SOURCE_METRICS) && get(PULSAR_STATS_INTERVAL_SECONDS) > 0;
```

Comment:
```suggestion
                get(PULSAR_ENABLE_SOURCE_METRICS) && get(PULSAR_STATS_INTERVAL_SECONDS) > 0;
```
?

Reply:
Yep, I rename the enum name from `PULSAR_DISABLE_SOURCE_METRICS` to `PULSAR_ENABLE_SOURCE_METRICS` and forget to change this assertion.